pgqd logs to syslog by default, so logfile is unnecessary.
We don't want pgqd daemonize itself, so pidfile is unnecessary.

--- a/pgqd.ini.dist
+++ b/pgqd.ini.dist
@@ -1,10 +1,7 @@
 [pgqd]
 
-# where to log
-logfile = ~/log/pgqd.log
-
-# pidfile
-pidfile = ~/pid/pgqd.pid
+# file path where to write logs
+#logfile =
 
 ## optional parameters ##
 
